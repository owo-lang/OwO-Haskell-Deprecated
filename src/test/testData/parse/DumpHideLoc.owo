module A where
 f : F (Type level)
 f = (a b) c (d e)