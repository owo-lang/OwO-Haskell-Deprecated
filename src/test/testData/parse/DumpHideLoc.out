File type: CodeFileType
Top module name: .A
 type signature "f"
  no pragmas
  function application
  function being applied
   named reference "F"
  value applied to the function
   function application
   function being applied
    named reference "Type"
   value applied to the function
    named reference "level"
 implementation of function  "f"
  no pragmas
  clause for  "f"
  patterns
  function body
   function application
   function being applied
    function application
    function being applied
     function application
     function being applied
      named reference "a"
     value applied to the function
      named reference "b"
    value applied to the function
     named reference "c"
   value applied to the function
    function application
    function being applied
     named reference "d"
    value applied to the function
     named reference "e"
  no with abstraction
  no where clause
